backend/tests/functional_tests/scripts/test_sync_configuration.py

```diff
@@ -448,9 +478,7 @@ def test_sync_with_changed_dataset_and_reused_task(prepare_environment: Any) ->
     """)
     script.sync_configuration(configuration)
 
-    # Check if Datasets were synchronized properly
-    datasets = DatasetsRepository.get_all_datasets(include_disabled=True)
```

Comment:
Why did you remove this check? Something has changed or was failing?

Reply:
No, I have no recollection of deleting these lines. It must have been done by mistake :smile: